dev/com.ibm.websphere.appserver.features/visibility/private/com.ibm.websphere.appserver.concurrent.mp-1.0.feature

```diff
@@ -2,8 +2,17 @@
 symbolicName=com.ibm.websphere.appserver.concurrent.mp-1.0
 visibility=private
 singleton=true
+-features=com.ibm.websphere.appserver.containerServices-1.0, \
+ com.ibm.websphere.appserver.appLifecycle-1.0, \
+ com.ibm.websphere.appserver.concurrencyPolicy-1.0, \
+ com.ibm.websphere.appserver.concurrent.mp-0.0.0.noImpl; ibm.tolerates:=1.0, \
```

Comment:
Some of the changes to this file are very confusing.  Do you have the reasoning behind why they are being made.
On this particular line, it seems like the com.ibm.websphere.appserver.concurrent.mp-1.0 feature is including an older version of itself and then tolerating the newest version of itself?  Are all of these changes needed?  If not, can the ones where are not needed be omitted?

Reply:
Looks like I only need com.ibm.websphere.appserver.contextService-1.0.  If I remove com.ibm.websphere.appserver.contextService-1.0 I get:  E **CWWKE0702E**: Could not resolve module: com.ibm.ws.concurrent.mp.1.0 [120]

Thanks for all of the great comments!